packages/unit-testing/jest-configuration/SuiteCloudJestConfiguration.js

```diff
@@ -13,30 +13,242 @@ const PROJECT_TYPE = {
 };
 
 const CORE_STUBS = [
+	{
+		module: 'N/action',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/action/action.js`,
+	},
+	{
+		module: 'N/action/instance',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/action/ActionInstance.js`,
+	},
+	{
+		module: 'N/auth',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/auth/auth.js`,
+	},
+	{
+		module: 'N/cache',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/cache/cache.js`,
+	},
+	{
+		module: 'N/cache/instance',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/cache/CacheInstance.js`,
+	},
+	{
+		module: 'N/certificateControl',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/certificateControl/certificateControl.js`,
+	},
+	{
+		module: 'N/certificateControl/certificate',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/certificateControl/Certificate.js`,
+	},
+	{
+		module: 'N/commerce/recordView',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/commerce/recordView.js`,
+	},
+	{
+		module: 'N/commerce/promising',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/commerce/promising.js`,
+	},
+	{
+		module: 'N/commerce/webstore/order',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/commerce/webstore/order.js`,
+	},
+	{
+		module: 'N/commerce/webstore/shopper',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/commerce/webstore/shopper.js`,
+	},
+	{
+		module: 'N/commerce/webstore/shopper/instance',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/commerce/webstore/ShopperInstance.js`,
+	},
+	{
+		module: 'N/compress',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/compress/compress.js`,
+	},
+	{
+		module: 'N/compress/archiver',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/compress/Archiver.js`,
+	},
+	{
+		module: 'N/crypto',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/crypto/crypto.js`,
+	},
+	{
+		module: 'N/crypto/certificate',
```

Comment:
What do you think about creating a "certificate" folder inside crypto for all these modules inside certificate?

Reply:
I would like to keep the flat structure as much as possible and avoid complications

Reply:
Well, I think this helps organize the modules.

Reply:
Done